build-logic/src/main/kotlin/Mpp.kt

```diff
@@ -122,6 +120,8 @@ fun Project.configureMppTestsDefaults(withJs: Boolean = true) {
     configureAppleTargets("macosX64", "macosArm64")
 
     addTestDependencies(withJs)
+
+    enableNewMemoryManager()
```

Comment:
Can we only enable it for defer tests? It's important to keep testing the old memory model if we continue maintaining it. (And I think we should given that [some users reported pretty bad performance at the moment](https://twitter.com/agLuque/status/1519408400907522051))

Reply:
Good point! I've enabled it only on the project (+ added a small readme to make it more visible).